### ISSUE TITLE
Support for write-protecting disk images

### ISSUE THREAD

**Issue by FrodeSolheim** [FrodeSolheim/fs-uae#24]
Currently, there is no way to write-protect disk images. Disk image overlay files are used, so whether the original file is write-protected or not does not affect the write-protected status in emulation.

**Comment by higgipoker** [comment]
Also encountered this problem with Championship Manager 93/94 update disks. the disk must be write protected to complete installation of the update.

**Comment by larss** [comment]
Sorry to bother, but is there any news on this front?
Thanks!

**Comment by FrodeSolheim** [comment]
No, not yet. The upcoming 2.8.0 will likely be released without proper support for this.
